backend/agent_runner/sandbox.py

```
@@ -0,0 +1,876 @@
+                         pr_url = text.split("__SOULCASTER_PR_URL__=", 1)[1].split('\n')[0].strip()
+                         if pr_url:
+                             await asyncio.to_thread(update_job, job.id, pr_url=pr_url, updated_at=datetime.now(timezone.utc))
+                             await asyncio.to_thread(
+                                 update_cluster,
+                                 str(job.project_id),
+                                 job.cluster_id,
+                                 github_pr_url=pr_url,
+                                 updated_at=datetime.now(timezone.utc),
+                             )
+                     except Exception:
+                         pass
```

Ruff (0.14.8)

`try`-`except`-`pass` detected, consider logging the exception

(S110)